terraform-docs.sh

```diff
@@ -1,16 +1,18 @@
 #!/bin/bash
 
 # https://stackoverflow.com/questions/59895/get-the-source-directory-of-a-bash-script-from-within-the-script-itself/246128#246128
-DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+AWK_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+DIR="$( cd "$2" >/dev/null 2>&1 && pwd )"
```

Comment:
Hmm, we actually `cd $DIR` here, effectively, first... So what is $DIR when $2 is a problem? Maybe do a little testing on the arguments first thing, before any of this. for example:

```
test -d $2 || ( echo "$2 must be a directory"; exit 1 )
```

I'd maybe recommend switching the thought process here some, too. It's more common for `DIR` to refer to this script's directory (as it did before this patch). Rather than changing that, use a different variable for this argument...

```
MODULE="$( cd "$2" >/dev/null 2>&1 && pwd )"
```

Reply:
Updated